dags/atd_knack_detectors.py

```diff
@@ -65,24 +64,22 @@
     dag_id="atd_knack_detectors",
     description="Load detectors (view_1333) records from Knack to Postgrest to AGOL and Socrata",
     default_args=DEFAULT_ARGS,
-    schedule_interval="10 9 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
+    schedule_interval="10 4 * * *" if DEPLOYMENT_ENVIRONMENT == "production" else None,
```

Comment:
hold my hand thru this cron syntax... run every day at 4:10 am?

Reply:
yes! I moved the time earlier because Christina / the MMC asked for it to happen overnight